lib/Jit/options.cpp

```diff
@@ -48,9 +48,7 @@ void freeStringConfigValue(ICorJitInfo *CorInfo, UTF16CharT *Value) {
   return CorInfo->freeStringConfigValue((wchar_t *)Value);
 }
 
-Options::Options(LLILCJitContext *Context) : Context(Context) {}
-
-void Options::initialize() {
+JitOptions::JitOptions(LLILCJitContext &Context) : Context(Context) {
```

Comment:
I've refactored the code to avoid stashing it away.  I'm still pretty convinced that I'm going to have to undo this in my next round of changes but it's undone.  (it's also now static)